Jenkinsfile

```diff
@@ -519,26 +419,21 @@ pipeline {
             git tag "${K_RELEASE_TAG}" "${release_commit}"
             git push origin "${K_RELEASE_TAG}"
 
-            LOCAL_BOTTLE_NAME=$(find ../big_sur -name "kframework--${VERSION}.big_sur.bottle*.tar.gz")
-            BOTTLE_NAME=$(echo ${LOCAL_BOTTLE_NAME#../big_sur/} | sed 's!kframework--!kframework-!')
-
             mv ../kframework-${VERSION}-src.tar.gz                      kframework-${VERSION}-src.tar.gz
             mv ../bionic/kframework_${VERSION}_amd64.deb                kframework_${VERSION}_amd64_bionic.deb
             mv ../focal/kframework_${VERSION}_amd64.deb                 kframework_${VERSION}_amd64_focal.deb
             mv ../bullseye/kframework_${VERSION}_amd64.deb              kframework_${VERSION}_amd64_bullseye.deb
             mv ../arch/kframework-git-${VERSION}-1-x86_64.pkg.tar.zst   kframework-git-${VERSION}-1-x86_64.pkg.tar.zst
-            mv $LOCAL_BOTTLE_NAME                                       $BOTTLE_NAME
 
             echo "K Framework Release ${VERSION}"  > release.md
             echo ''                               >> release.md
             cat k-distribution/INSTALL.md         >> release.md
-            hub release create                                                                  \
+            hub release create --prerelease                                                     \
```

Comment:
I want to make sure I understand how this works. So this release will be cut, and then the Mac package will be triggered (because of the `prereleased` tag) _after_ this release is cut, and it will turn it into a proper release?

Reply:
So there is a chance that mac release fails, which leads to the release staying as a pre-release?

Reply:
if the mac release fails, the github workflow will delete the release.